src/App.tsx

```diff
@@ -25,7 +26,8 @@ function App () {
       <ThemeProvider theme={theme}>
         <ErrorHandler>
           <AuthenticationProvider>
-            <RouterProvider router={router} />
+              <Header />
```

Comment:
The header can be set in the router, (see oss app for an example). Doing this will then mean that in the authentication-provider children can be typed as JSX.Element rather than JSX.Element[]

Reply:
Done (hopefully)